src/components/BookingForm/BookingForm.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { createNewVespa, fetchVespas } from '../../redux/slices/vespaSlice';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './BookingForm.css';
+
+const BookingForm = () => {
```

Comment:
- [ ] In the "Book a Vespa" section, this should read "End date".  Kindly update this text for consistency:

![image](https://github.com/Rudy-Menekam/Rentals-project-frontend/assets/57421823/d07b60db-f2c7-4af3-b3de-94afcadfc109)